apps/kol-amelamdim/pages/category/[category].tsx

```diff
@@ -0,0 +1,174 @@
+import {
+  useEffect,
+  useState,
+  ChangeEvent,
+  useContext,
+  ReactElement,
+} from 'react';
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  Paper,
+  TablePagination,
+  Link,
+  Button,
+} from '@mui/material';
+import { useRouter } from 'next/router';
+import { Category, IFile } from '@kol-amelamdim/types';
+import { StyledPageContainer } from '@kol-amelamdim/styled';
+import { FILE_TYPES_DICTIONARY } from '@kol-amelamdim/constants';
+import { API_ERRORS } from '@kol-amelamdim/api-errors';
+import { FilterCard } from '../../components/filter-card/FilterCard';
+import { useCategoriesFiles } from '../../hooks/use-categories-files';
+import { AuthContext } from '../../context/auth-context-provider';
+import { UploadFileDialog } from '../../components';
+import { AlertContext } from '../../context/alert-context-provider';
+import { AlertLayout } from '../../layouts';
+
+const Mivhanim = () => {
+  const [fileType, setFileType] = useState('');
+  const [filterText, setFilterText] = useState('');
+  const [page, setPage] = useState<number>(0);
+  const [rowsPerPage, setRowsPerPage] = useState<number>(5);
```

Comment:
I think the initial should be 25 and it's not need to be changed. 
The reason is that for my opinion if we are showing 5 rows then we don't really need the filter in some cases.

Reply:
Resolved!